build/build.js

```diff
@@ -42,6 +43,7 @@ Program
   .option('--compile-js, --compile-js path', 'Handles ES6, ES5 and web component scripts')
   .option('--compile-css, --compile-css path', 'Compiles all the scss files to css')
   .option('--watch', 'Watch file changes and re-compile (ATM only works for the js in the media_source).')
+  .option('--gzip', 'Precompress all the minified stylesheets and scripts.')
```

Comment:
Not quite sure what the correct word is but its definitely not precompress. Pre- is something that takes before.

```suggestion
  .option('--gzip', 'Compress all the minified stylesheets and scripts.')
```